corner_detector:
    laser_topic: "/amigo/base_laser/scan"
    sensor_noise_std_dev: 0.05
    corner_threshold: 0.08
    step_size: 0.2
    frame_id: "/amigo/base_laser"
    jump_size: 0.10
    visualization:
        lifetime: 0.1
        points:
            name: "corners"
            color: {r: 0, g: 1, b: 0}
        lines:
            name: "corner_lines"
            color: {r: 0, g: 1, b: 1}
association:
    association_modules:
      - type: "nearest_neighbor"
        max_no_std_devs: 0.2
      - type: "edge_tension"
        max_no_std_devs: 2.0
odom_tracker:
    odom_frame: "amigo/odom"
    sensor_frame: "amigo/base_laser"
graph_filename: "test_area_1.yaml"
initial_pose:
    - position: {x: 2.5, y: 1.2}
      node: 5
      std_dev: 0.10
    - position: {x: 2.5, y: -1.2}
      node: 2
      std_dev: 0.10
visualization:
    lifetime: 0.1
    points:
        name: "graph_nodes"
        color: {r: 1, g: 0, b: 0}
    lines:
        name: "graph_edges"
        color: {r: 1, g: 1, b: 0}
